README.md

```diff
@@ -17,26 +17,29 @@ Connecting to an already running runtime
 ```javascript
 const { connect, Identity } = require("hadouken-js-adapter");
 
-connect({
-    address: "ws://localhost:9696",
-    uuid: "my-uuid-123"
-}).then(logic).catch(connError);
-
-function logic(fin) {
-
-    fin.System.getVersion().then(v => console.log("Connected to Hadouken version", v));
-    let win;
-    fin.Window.wrap({
-        uuid: "remote-app-uuid",
-        name: "remote-window-name"
-    }).then(w => { win = w; win.moveBy(500, 0)).then(win.flash());
-
+async function launchApp() {
```

Comment:
Do we want async functions on Node 6.9?

Reply:
stable node is v9, think it's ok by now.